src/logic/safe/utils/safeInformation.ts

```diff
@@ -6,11 +6,14 @@ import { sameAddress } from 'src/logic/wallets/ethAddresses'
 import { SafeRecordProps } from '../store/models/safe'
 import { SafeRecordWithNames } from '../store/selectors'
 
+const GATEWAY_ERROR = /1337|42/
+
 export const getSafeInfo = async (safeAddress: string): Promise<SafeInfo> => {
   try {
     return await fetchSafeInfo(getClientGatewayUrl(), getNetworkId().toString(), safeAddress)
   } catch (e) {
-    throw new CodedException(Errors._605, e.message)
+    const safeNotFound = GATEWAY_ERROR.test(e.message || '')
```

Comment:
Can't you just test against `e?.message`?

Reply:
Yes, actually `e.message` w/o `?` would also work.